sources/packages/web/src/components/common/students/StudentApplicationsSimplifiedSummary.vue

```diff
@@ -0,0 +1,162 @@
+<!-- This component is shared between ministry and institution users. -->
+<template>
+  <body-header-container>
+    <template #header>
+      <body-header
+        title="Applications"
+        :recordsCount="applicationsAndCount.count"
+      ></body-header>
+    </template>
+    <content-group>
+      <toggle-content
+        :toggled="!applicationsAndCount.results?.length"
+        message="No applications are currently available."
+      >
+        <v-data-table-server
+          :headers="StudentApplicationsSimplifiedSummaryHeaders"
+          :items="applicationsAndCount.results"
+          :items-per-page="DEFAULT_PAGE_LIMIT"
+          :items-per-page-options="ITEMS_PER_PAGE"
+          :items-length="applicationsAndCount.count"
+          :loading="loading"
+          :mobile="isMobile"
+          @update:options="paginationAndSortEvent"
+        >
+          <template #[`item.applicationNumber`]="{ item }">
+            {{ item.applicationNumber }}
+          </template>
+          <template #[`item.submitted`]="{ item }">
+            {{
+              emptyStringFiller(getISODateHourMinuteString(item.submittedDate))
+            }}
+          </template>
+          <template #[`item.studyStartPeriod`]="{ item }">
+            {{
+              dateOnlyLongPeriodString(
+                item.studyStartPeriod,
+                item.studyEndPeriod,
+              )
+            }}
+          </template>
+          <template #[`item.status`]="{ item }">
+            <status-chip-application :status="item.status" />
+          </template>
+          <template #[`item.actions`]="{ item }">
+            <v-btn
+              color="primary"
+              variant="outlined"
+              @click="$emit('goToApplication', item.id)"
+              >View</v-btn
+            >
+          </template>
+        </v-data-table-server>
+      </toggle-content>
+    </content-group>
+  </body-header-container>
+</template>
+
+<script lang="ts">
+import { onMounted, ref, defineComponent } from "vue";
+import {
+  DEFAULT_PAGE_LIMIT,
+  DataTableSortOrder,
+  StudentApplicationFields,
+  StudentApplicationsSimplifiedSummaryHeaders,
+  ITEMS_PER_PAGE,
+  DataTableOptions,
+  PaginationOptions,
+} from "@/types";
+import { ApplicationService } from "@/services/ApplicationService";
+import { useFormatters } from "@/composables";
+import StatusChipApplication from "@/components/generic/StatusChipApplication.vue";
+import {
+  ApplicationSummaryAPIOutDTO,
+  PaginatedResultsAPIOutDTO,
+} from "@/services/http/dto";
+import { useDisplay } from "vuetify";
+
+export default defineComponent({
+  components: { StatusChipApplication },
+  emits: {
+    goToApplication: (applicationId: number) => {
+      return !!applicationId && applicationId > 0;
+    },
+  },
+  props: {
+    studentId: {
+      type: Number,
+      required: false,
+    },
```

Comment:
Do we require studentId prop in simplified summary?

Reply:
The simplified is where we need it because will be consumed by Ministry/Institution.
The extended can have it removed.

Reply:
Changed in the student component.